components/Reviews/components/AllReviewsLink/style.js

```diff
@@ -9,11 +9,23 @@ import { css } from 'glamor';
 import colors from 'Styles/colors';
 import variables from 'Styles/variables';
 
-export default css({
+/**
+ * Styles for a link parent container.
+ * @type {string}
+ */
+export const containerStyle = css({
```

Comment:
No named exports in styles please.

Reply:
Also please don't prefix everything with `Style`.

Reply:
done